src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar
+
+class Car(
+    private var location: Int = 0,
+) {
+    fun move(randNumber: Int) {
+        val rule = MovingJudgeRule.judge(randNumber)
+        val strategy = rule.strategy()
+
+        location = strategy.move(location)
```

Comment:
strategy가 직접 location을 변경하는 것이 아니라, 아래와 같이 변경하거나 변경되는 값만큼 기존 상태에서 추가해주는 것이 더 좋을 것 같아요. 🤔

```suggestion
        val strategy = rule.strategy()
        if(strategy.isMovable()) {
            location++
        }
```